src/main/resources/theme/groups/account/src/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "lib": [ "es2015", "dom", "dom.iterable" ],
-    "noImplicitAny": true,
```

Comment:
What this change means?

Reply:
if it's enabled, whenever you have a variable definition, you need to set its type. i.e. if you have a function function1(param1: any) with parameter param1, you always need to state its type or you will get a compilation error (typescript compiler).
If you disable, you can just let the compiler infer the "any" type whenever you don't define the type, without any whining... 
See here:
https://www.typescriptlang.org/tsconfig#noImplicitAny

Reply:
Thanks